webrtc.html

```diff
@@ -6366,24 +6366,21 @@ <h2>
                 </dd>
                 <dt class="add-to-usernameFragment-desc">
                   <dfn>relayProtocol</dfn> of type <span class=
-                  "idlMemberType">RTCIceServerTransportProtocol</span>
+                  "idlMemberType">RTCIceServerTransportProtocol</span>, readonly, nullable
                 </dt>
                 <dd class="add-to-usernameFragment-desc">
-                  <p>
-                    The protocol used by the endpoint to communicate with the TURN server. This
-                    is only present for local relay candidates.
-                  </p>
+                  For local relay candidates this is the protocol used by the endpoint to
```

Comment:
for consistency of language, should this be "for local candidates of type {{relay}}"?
someone could interpret the difference in markup between the two sections as meaning that there are other candidates of type relay.